op-e2e/external_geth/main.go

```diff
@@ -46,10 +46,10 @@ func run(configPath string) error {
 
 	binPath, err := filepath.Abs("op-geth")
 	if err != nil {
-		return fmt.Errorf("could not get absolute path of op-geth")
+		return errors.New("could not get absolute path of op-geth")
```

Comment:
It should be the same as `op-e2e/external_geth/main.go` line 90, use errors.New to replace fmt.Errorf with no parameters.